DemoApp/AppConfigViewController/SwitchButton.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+class SwitchButton: UISwitch {
+    var didChangeValue: ((Bool) -> Void)?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addTarget(self, action: #selector(didChangeValue(sender:)), for: .valueChanged)
```

Comment:
Won't this cause a memory leak?

Reply:
This in particular is not causing memory leaks, on the other hand, forgot to add `weak` on closures using this SwicthButton. Fixed that, run the memory graph and we don't have anymore memory leaks 👍 ✅